src/components/v5/shared/Button/Button.module.css

```diff
@@ -38,6 +38,10 @@
   @apply bg-base-white text-gray-700 border border-gray-200 md:hover:bg-blue-400 md:hover:text-base-white md:hover:border-blue-400 disabled:text-gray-300;
 }
 
+.socialLinkType {
```

Comment:
This name is related to a place where this variant is used. Please make it more reusable.

Reply:
I can't see any changes here